src/containers/BooksForm.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+class BookForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { title: '', category: 'Action' };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const { props: { createBook }, state: { title, category } } = this;
+    const book = {
+      id: Math.random().toString(),
+      title,
+      category,
+    };
+    createBook(book);
+    this.setState({ title: '' });
+  }
```

Comment:
Let's reset the component state here, this includes the form data.

You can use [`HTMLFormElement.reset()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset) here.

Note that the initial state is `this.state = { title: '', category: 'Action' };`


> implements `handleSubmit` and attaches it to form's `onSubmit` event [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleSubmit.png)